services/related.py

```
@@ -369,3 +388,78 @@ def _compute_recency_decay(self, current_date: date, post_date: date) -> float:
+                    post_data = {
+                        "title": frontmatter.get("title", ""),
+                        "date": post_date,
+                        "tags": frontmatter.get("tags", []),
+                        "description": frontmatter.get("description", ""),
+                        "url": f"https://paulchrisluke.com/blog/{post_date}",
+                        "digest": digest
+                    }
+                    
+                    posts.append(post_data)
+                    
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)